content/en/docs/reference/_index.md

```diff
@@ -19,7 +20,19 @@ This section of the Kubernetes documentation contains references.
 ## API Reference
 
 * [Kubernetes API Reference {{< latest-version >}}](/docs/reference/generated/kubernetes-api/{{< latest-version >}}/)
-* [Using The Kubernetes API](/docs/reference/using-api/) - overview of the API for Kubernetes.
+* [Kubernetes API Overview](/docs/reference/using-api/) - overview of the API for Kubernetes.
```

Comment:
@raghvenders , Are these the section to add (keep the text to match the page titles):

Standardized Glossary
Kubernetes Issues and Security
API Access Control
Scheduling
Tools (single page)

Currently, `Scheduling` is listed under the Component Reference section on this page as:

```
 * [kube-scheduler Policies](/docs/reference/scheduling/policies)
  * [kube-scheduler Profiles](/docs/reference/scheduling/config#profiles)
```
Perhaps these links could be pulled out and put under a `Scheduling` heading?

Reply:
This makes sense, I will add the changes.

Reply:
@kbhawkey  - addressed this review comments